<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EpgShow</class>
 <widget class="QWidget" name="EpgShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset theme="x-office-calendar">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="layoutMain">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="header">
        <item>
         <layout class="QVBoxLayout" name="labels">
          <item>
           <widget class="QLabel" name="labelTitle">
            <property name="text">
             <string notr="true">Title</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="info">
            <item>
             <widget class="QLabel" name="labelInfoIcon">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelInfo">
              <property name="text">
               <string notr="true">Info</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="time">
            <item>
             <widget class="QLabel" name="labelTimeIcon">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelTime">
              <property name="text">
               <string notr="true">Time</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="channel">
            <item>
             <widget class="QLabel" name="labelChannelIcon">
              <property name="minimumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>22</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelChannel">
              <property name="text">
               <string notr="true">Channel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelPhoto">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">Photo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="close">
          <item>
           <widget class="QPushButton" name="buttonClose">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="icon">
             <iconset theme="window-close">
              <normaloff/>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBlank">
            <property name="maximumSize">
             <size>
              <width>1</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="content">
        <item>
         <layout class="QGridLayout" name="layoutDescription">
          <item row="0" column="0">
           <widget class="QLabel" name="labelDescriptionIcon">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>285</width>
               <height>215</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="labelDescription">
                <property name="text">
                 <string notr="true">Description</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacerDescription">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelDescriptionName">
            <property name="text">
             <string>&lt;b&gt;Description&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="layoutCrew">
          <item row="0" column="0">
           <widget class="QLabel" name="labelCrewIcon">
            <property name="minimumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelCrew">
            <property name="text">
             <string>&lt;b&gt;Crew&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QListView" name="crew">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QComboBox" name="comboCrewType">
            <item>
             <property name="text">
              <string>Crew members: All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Director</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Actor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Writer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adapter</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Producer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Composer</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Editor</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Presenter</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Commentator</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Guest</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttons">
        <item>
         <widget class="QPushButton" name="buttonPrevious">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Previous</string>
          </property>
          <property name="icon">
           <iconset theme="go-previous">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerButtons1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonRecord">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Record</string>
          </property>
          <property name="icon">
           <iconset theme="media-record">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerButtons2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNext">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Next</string>
          </property>
          <property name="icon">
           <iconset theme="go-next">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
